include/flatbuffers/grpc.h

```diff
@@ -38,13 +38,17 @@ class Message {
   Message(grpc_slice slice, bool add_ref)
     : slice_(add_ref ? grpc_slice_ref(slice) : slice) {}
 
-  Message &operator=(const Message &other) = delete;
+  Message(const Message &other) : slice_(grpc_slice_ref(other.slice_)) {}
 
   Message(Message &&other) : slice_(other.slice_) {
     other.slice_ = grpc_empty_slice();
   }
 
-  Message(const Message &other) = delete;
+  Message &operator=(const Message &other) {
+    grpc_slice_unref(slice_);
```

Comment:
documentation for this function says that it only works on initialized slices. Does that mean it works on a `grpc_empty_slice()` ?